packages/flutter/test/foundation/isolates_test.dart

```diff
@@ -29,4 +33,23 @@ void main() {
     expect(await compute(test1Async, 0), 1);
     expect(compute(test2Async, 0), throwsException);
   }, skip: kIsWeb);
+
+  test('compute closes all ports', () async {
+    // Run a Dart script that calls compute().
+    // The Dart process will terminate only if the script exits cleanly with
+    // all isolate ports closed.
+    const FileSystem fs = LocalFileSystem();
+    const Platform platform = LocalPlatform();
+    final String flutterRoot = platform.environment['FLUTTER_ROOT']!;
+    final String dartPath = fs.path.join(flutterRoot, 'bin', 'cache', 'dart-sdk', 'bin', 'dart');
+    final String packageRoot = fs.path.dirname(fs.path.fromUri(platform.script));
+    final String scriptPath = fs.path.join(packageRoot, 'test', 'foundation', '_compute_caller.dart');
+    final Future<ProcessResult> resultFuture = Process.run(dartPath, <String>[scriptPath]);
+    final ProcessResult? result = await Future.any(<Future<ProcessResult?>>[
+      resultFuture,
+      Future<ProcessResult?>.delayed(const Duration(seconds: 5), null),
```

Comment:
When CI is under heavy load this might not be enough time. I think this is probably a case where we want to time out faste rthan the test timeout, but it should probably be a few minutes to be safe rather than a few seconds. Then again, the style guide says to just not use timeouts like this so maybe we should just let the test timeout be it.

Reply:
Actually I'm pretty sure we don't override the default tester timeout of 30s for these types of tests right now, which should be sufficient.

Reply:
done